charts/aws-fsx-csi-driver/templates/csidriver.yaml

```diff
@@ -3,4 +3,12 @@ kind: CSIDriver
 metadata:
   name: fsx.csi.aws.com
 spec:
-  attachRequired: false
+  attachRequired: {{ .Values.csidriver.attachRequired }}
+  fsGroupPolicy: {{ .Values.csidriver.fsGroupPolicy }}
```

Comment:
This one is an exception, I think. `ReadWriteOnceWithFSType` effectively means that kubelet won't change ownership of the files. Changing this to `File` will cause kubelet to change owner every time this is mounted, and if mounted on many nodes at the same time can lead to interesting situations such as deadlocks.

Its however fairly safe _if_ every mounting Pod has the same `securityContext` _and_ it includes `fsGroupChangePolicy: "OnRootMismatch"`

We may want to document this somewhere.

Reply:
I agree. For now, I put it in the `changelog.md` file, but this is not ok. Maybe it would make sense to create a `README.md` for the helm chart? 

Also: removed the configuration items except `fsGroupPolicy`